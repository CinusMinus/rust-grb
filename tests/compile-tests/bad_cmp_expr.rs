use gurobi::*;
mod utils;

fn main() -> Result<()> {
  let _g = gag::Gag::stdout();
  create_model!(_g, m, x, y, z);
  c!(x + y == );
  c!(==);
  c!(x);
  c!(>=3);
  Ok(())
}
