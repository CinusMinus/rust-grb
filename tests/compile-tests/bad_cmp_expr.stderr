error: unexpected end of input, expected expression
 --> $DIR/bad_cmp_expr.rs:7:3
  |
7 |   c!(x + y == );
  |   ^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected expression
 --> $DIR/bad_cmp_expr.rs:8:6
  |
8 |   c!(==);
  |      ^^

error: expected binary operation
 --> $DIR/bad_cmp_expr.rs:9:6
  |
9 |   c!(x);
  |      ^

error: expected expression
  --> $DIR/bad_cmp_expr.rs:10:6
   |
10 |   c!(>=3);
   |      ^^
